opencti-platform/opencti-front/src/private/components/settings/notifications/notifier/NotifierLine.tsx

```diff
@@ -0,0 +1,162 @@
+import Chip from '@mui/material/Chip';
+import ListItem from '@mui/material/ListItem';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import Skeleton from '@mui/material/Skeleton';
+import makeStyles from '@mui/styles/makeStyles';
+import React, { FunctionComponent } from 'react';
+import { graphql, useFragment } from 'react-relay';
+import ItemIcon from '../../../../../components/ItemIcon';
+import { DataColumns } from '../../../../../components/list_lines';
+import { Theme } from '../../../../../components/Theme';
+import { NotifierLine_node$key } from './__generated__/NotifierLine_node.graphql';
+import { NotifierLinesPaginationQuery$variables } from './__generated__/NotifierLinesPaginationQuery.graphql';
+import NotifierPopover from './NotifierPopover';
+
+const useStyles = makeStyles<Theme>((theme) => ({
+  item: {
+    paddingLeft: 10,
+    height: 50,
+  },
+  itemIcon: {
+    color: theme.palette.primary.main,
+  },
+  bodyItem: {
+    height: 20,
+    fontSize: 13,
+    float: 'left',
+    whiteSpace: 'nowrap',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    paddingRight: 5,
+  },
+  goIcon: {
+    position: 'absolute',
+    right: -10,
+  },
+  itemIconDisabled: {
+    color: theme.palette.grey?.[700],
+  },
+  placeholder: {
+    display: 'inline-block',
+    height: '1em',
+    backgroundColor: theme.palette.grey?.[700],
+  },
+  drawerPaper: {
+    minHeight: '100vh',
+    width: '50%',
+    position: 'fixed',
+    overflow: 'auto',
+    transition: theme.transitions.create('width', {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.enteringScreen,
+    }),
+    padding: 0,
+  },
+  chipInList: {
+    fontSize: 12,
+    height: 20,
+    float: 'left',
+    width: 120,
+  },
+}));
+
+interface NotifierLineProps {
+  node: NotifierLine_node$key;
+  dataColumns: DataColumns;
+  paginationOptions: NotifierLinesPaginationQuery$variables
+}
+
+const NotifierLineFragment = graphql`
+  fragment NotifierLine_node on Notifier {
+    id
+    entity_type
+    name
+    description
+    notifier_connector {
+      name
+    }
+  }
+`;
+
+export const NotifierLine: FunctionComponent<NotifierLineProps> = ({
+  dataColumns,
+  node,
+  paginationOptions,
+}) => {
+  const classes = useStyles();
+  const data = useFragment(NotifierLineFragment, node);
+  return (
+    <>
+      <ListItem classes={{ root: classes.item }} divider={true} button={true}>
```

Comment:
The line is clickable but nothing happen

```suggestion
      <ListItem classes={{ root: classes.item }} divider={true}>
```